index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Imperial Donuts</title>
+</head>
+<body>
+    <header>
+        
```

Comment:
please don't add blank lines without a reason.

Reply:
- [ ] I suggest you explain why this change is required, that will make it very easy to implement the changes because the one who will read it will know the context

Reply:
please don't add blank lines without a reason. Unresenoble blank lines will make your project ambiguous and will make you look unprofesional.

Reply:
good job 👍